docs/source/related.rst

```diff
@@ -0,0 +1,75 @@
+Related efforts and solutions
+*****************************
+
+To our knowledge, there is no other effort with as broad of a scope as datalad
+that aims to unify access to vast arrays of scientific data in a domain and
+data modality agnostic fashion with as few and universally available software
+dependencies.
+
+The most comparable project regarding the idea of federating access to various
+data providers is the iRODS_-based `INCF Dataspace`_ project.  IRODS is a
+powerful, NSF-supported framework, but it requires non-trivial deployment and
+management procedures. As a representative of *data grid* technology, it is
+more suitable for an institutional deployment, as data access, authentication,
+permission management, and versioning are complex and not-feasible to be
+performed directly by researchers. Datalad on the other hand federates
+institutionally hosted data, but in addition enables individual researchers and
+small labs to contribute datasets to the federation with minimal cost and
+without the need for centralized coordination and permission management.
+
+.. _IRODS: https://irods.org
+.. _INCF Dataspace: http://www.incf.org/resources/data-space
```

Comment:
well, since then there came git-lfs, lpfs, but I don't think we should aim to compare to all of them since, git-annex has already a page on that:  http://git-annex.branchable.com/not/  so might be worth linking to it here or in git-annex section.